src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Deciders/BankAccount.fs

```diff
@@ -0,0 +1,69 @@
+﻿module Deciders.BankAccount
+
+open System
+open BankAccountPrimitives
+
+type BankAccountOpened =
+    { BankAccountId: AccountId
+      AccountNumber: AccountNumber
+      ClientId: ClientId
+      CurrencyISOCode: CurrencyCode
+      CreatedAt: DateTimeOffset
+      Version: int64 }
+
+type DepositRecorded =
+    { BankAccountId: AccountId
+      Amount: decimal
+      CashierId: CashierId
+      RecordedAt: DateTimeOffset
+      Version: int64 }
+
+type CashWithdrawnFromATM =
+    { BankAccountId: AccountId
+      Amount: decimal
+      ATMId: AtmId
+      RecordedAt: DateTimeOffset
+      Version: int64 }
+
+type BankAccountClosed =
+    { BankAccountId: AccountId
+      Reason: string
+      ClosedAt: DateTimeOffset
+      Version: int64 }
+
+type Event =
+    | BankAccountOpened of BankAccountOpened
+    | DepositRecorded of DepositRecorded
+    | CashWithdrawnFromATM of CashWithdrawnFromATM
+    | BankAccountClosed of BankAccountClosed
+
+type BankAccount =
+    | NotInitialised
```

Comment:
It's just in it's `Initial` state - also lets you skip out of Initialized vs Initialised debates

Reply:
Good call. Naming is hard 😅 Changed in https://github.com/oskardudycz/Ogooreck/pull/13/commits/9c3bd4fa7efd66b75bf111513ba31eea4892e0cb.